app/controllers/api/v1/audios_controller.rb

```diff
@@ -115,7 +116,8 @@ def create
             { identifier: params[:flowIdentifier],
```

Comment:
@jonatasrancan Changing the line above from `AudioProcessing.perform_later` to `AudioProcessing.perform_now` fixes everything, and everything works. Any ideas why it's failing?! Perhaps somehow the full environment of the original execution does not get loaded?